crates/exex/src/model.rs

```diff
@@ -266,6 +297,14 @@ impl Default for KethPointer {
     }
 }
 
+impl KethEncodable for KethPointer {
+    fn encode(&self) -> Vec<MaybeRelocatable> {
+        let mut encoded = vec![self.len.0.clone()];
+        encoded.extend(self.data.iter().map(|item| item.0.clone()));
+        encoded
+    }
```

Comment:
not sure about the underlying type you target, I understand that this creates a single segment with `[len, ...data]` while it should probably be just `[len, data*]`

Reply:
This is because in my keth pointer I've:
- len: The length of the data to be stored
- data: a vector of MaybeRelocatable holding the main data

So here I simply group them all into a single vector of MaybeRelocatable to have something that I'll inject into gen_arg at the end